src/it/scala/net/snowflake/spark/snowflake/SnowflakeTelemetrySuite.scala

```diff
@@ -110,6 +120,7 @@ class SnowflakeTelemetrySuite extends IntegrationSuiteBase {
         .format(SNOWFLAKE_SOURCE_NAME)
         .options(connectorOptionsNoTable)
         .option("dbtable", test_table)
+        .mode(SaveMode.Overwrite)
```

Comment:
Impact of this change?

Reply:
It may be affected by the new test or something else.
I met test case failure because the save mode is `ErrorIfExists`. So, I update to use OverWrite mode.